content/en/service_catalog/adding_metadata.md

```diff
@@ -40,6 +40,22 @@ For more information about the latest updates, see the schemas on GitHub.
 ### Metadata Schema v3.0 (beta)
 The Entity Definition Schema is a structure that contains basic information about an entity.
 
+#### New features in v3.0 
+##### Expanded Analysis Units
+Beyond services, now you can organize your systems using various components such as applications, services, queues, datastores, and libraries (WIP). This holistic view supports better analysis and management.
```

Comment:
```suggestion
Beyond services, you can organize your systems using various components such as applications, services, queues, and datastores. This holistic view supports better analysis and management.
```
Datadog's public documentation must not reference time. Instead, we write what is true now. When you add support for libraries, you can add that word to the docs.